war/src/main/js/widgets/config/tabbar.less

```diff
@@ -1,85 +1,184 @@
 /*
  * Tab bar specific rules.
  */
+body.add-item.hide-side  #side-panel{width:0; padding:0; overflow:hidden; transition: all 0.5s ease;}
+body.add-item.hide-side  #main-panel{margin:0 auto; max-width:900px; transition: all 0.5s ease;}
+body.add-item .jenkins-config table tr {display:none;}
+body.add-item .jenkins-config table tr:last-of-type {display:table-row}
+body.add-item .jenkins-config-widgets .form-config.tabBarFrame .tabBar .tab.active {background:@bright; border-bottom-color:@bright}
+body.add-item .category.jenkins-config.hide-cat {display:none;}
+body#jenkins.add-item .jenkins-config {
```

Comment:
These are specific overrides to the general settings CSS as established in the config file.
...so it is easier to target the specific overrides on a line-by-line basis.

...that having been said, I can likely be more eficent with Less, so I will take another pass at it.